grpc/server/src/main/java/io/helidon/grpc/server/ProtoReflectionService.java

```diff
@@ -90,6 +92,8 @@ public final class ProtoReflectionService extends ServerReflectionGrpc.ServerRef
 
     private final Object lock = new Object();
```

Comment:
Random drive by comment: `lock` is unused now, isn't it?

Reply:
Definitely. Not sure why our style checking, etc. let this through. Thanks for the "drive-by" comment!